src/Microsoft.AspNetCore.Sockets/WebSockets.cs

```diff
@@ -2,143 +2,155 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Net.WebSockets;
-using System.Threading;
 using System.Threading.Tasks;
 using Channels;
 using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.WebSockets.Internal;
+using Microsoft.Extensions.Internal;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.WebSockets.Internal;
```

Comment:
Yes, Extensions has the IWebSocketConnection and AspNetCore has IHttpWebSocketConnectionFeature.